### ISSUE TITLE
The app .aab file is not loaded on the Android store, api level is not supported

### ISSUE THREAD

**Issue by websystemtechnology** [gluonhq/gluonfx-gradle-plugin#163]
When I try to upload my app's .aab file to the Android store, I get this error: "The current target API level of your app is 30, but the minimum target API level must be 31 to ensure that the app is based on the Latest APIs optimized for security and good performance. Change the target API level of your app to at least 31. "
Where can I change this level?
Neither on the build.gradle file nor on the AndroidManifest.xml file I find where to change it.

**Comment by ctoabidmaqbool** [comment]
@websystemtechnology You can use most of the normal Android related settings (e.g. `<uses-sdk android:minSdkVersion="21" android:targetSdkVersion="31"/>`) in Android Manifest file under `src/android/AndroidManifest.xml`:

e.g.

```
<?xml version="1.0" encoding="UTF-8"?>
<manifest xmlns:android="http://schemas.android.com/apk/res/android"
          package="com.gluonhq.hello"
          android:versionCode="2"
          android:versionName="2022.09.12">
	
    <uses-sdk android:minSdkVersion="21" android:targetSdkVersion="31"/>
    
    <supports-screens android:xlargeScreens="true"/>
    
    <uses-permission android:name="android.permission.INTERNET"/>
    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"/>
    <uses-permission android:name="android.permission.MANAGE_EXTERNAL_STORAGE"/>
    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
    <uses-permission android:name="android.permission.WAKE_LOCK"/>

    <application android:label="Hello Gluon App"
                 android:name="android.support.multidex.MultiDexApplication"
                 android:icon="@mipmap/ic_launcher">
        
        <activity android:name="javafxports.android.FXActivity"
                  android:label="Hello Gluon App"
                  android:exported="true"
                  android:configChanges="orientation|screenSize">
			
            <meta-data android:name="main.class" android:value="com.gluonhq.hello.HelloGluonApp"/>
            <meta-data android:name="preloader.class" android:value="com.gluonhq.hello.AppPreloader"/>
            <meta-data android:name="debug.port" android:value="0"/>

            <intent-filter>
                <action android:name="android.intent.action.MAIN"/>
                <category android:name="android.intent.category.LAUNCHER"/>
            </intent-filter>
            
        </activity>

        <activity android:name="com.gluonhq.impl.charm.down.plugins.android.PermissionRequestActivity" />
        
        <provider
            android:name="android.support.v4.content.FileProvider"
            android:authorities="com.gluonhq.hello.fileprovider"
            android:grantUriPermissions="true">
            <meta-data
                android:name="android.support.FILE_PROVIDER_PATHS"
                android:resource="@xml/file_provider_paths" />
        </provider>
        
    </application>
</manifest>
```

**Note:** It's old plugin file, somethings like that can be used in new gluon plugin too! You should modify as you needed it

**Comment by websystemtechnology** [comment]
I added in the AndroidManifest.xml file: 
   <uses-sdk android: minSdkVersion =" 21 "android: targetSdkVersion =" 31 "/>

but running the ./gradlew nativePackage fails:
   ./gradlew nativePackage
   [INFO] ==================== PACKAGE TASK ====================
   [SEVERE] Process package-task failed with result: 1
   [INFO] Logging process [package-task] to file: /h/e-ws/app/build/gluonfx/log/process-package-task-1664352794672.log
   [SEVERE] Packaging failed.

**Comment by jperedadnr** [comment]
Fixed with version 1.0.16-SNAPSHOT

**Comment by websystemtechnology** [comment]
the problem is still present
